README.md

```diff
@@ -6,9 +6,9 @@ Keeping the trash in your ./vendor dir to a minimum.
 
 Make sure you're using go1.6 or later version.
 
- 0. Download and extract `trash` to your PATH
+ 1. Download and extract `trash` to your PATH: `go get github.com/rancher/trash`.
```

Comment:
It's currently numbered starting at 0, so it seems they both render the same. @imikushin most markdown I've seen begins at 1. Any reason why we can't just switch to that?